lib/storage.dart

```diff
@@ -181,18 +181,18 @@ class Acl {
     return _entries.map((entry) => entry._toObjectAccessControl()).toList();
   }
 
+  @override
   int get hashCode {
-    return _cachedHashCode != null
-        ? _cachedHashCode
-        : _cachedHashCode = _jenkinsHash(_entries);
+    return _cachedHashCode ?? _cachedHashCode = _jenkinsHash(_entries);
```

Comment:
prefer_conditional_assignment or prefer_if_null_operators ?

This is invalid code!

This should be `return _cachedHashCode ??= _jenkinsHash(_entries);`

Or do nothing...happens two other times in this file